src/Model.elm

```diff
@@ -67,13 +67,18 @@ config =
         (Kinto.Basic "test" "test")
 
 
-resourceConfig =
+recordConfig recordId =
     Kinto.configureResource
         "https://kinto.dev.mozaws.net/v1/"
+        (Kinto.RecordEndpoint "default" "test-items" recordId)
+        (Kinto.Basic "test" "test")
+
+
+recordListConfig =
```

Comment:
I feel like the basic server config should be distinguished from the resource configuration (eg. here `recordConfig` and `recordListConfig` share a lot). And to be entirely honest, I also feel like users shouldn't have to deal with resource endpoint configuration at all.

Ideally as a lib user I'd like a slighly higher level API allowing to simply request for resources using dedicated functions where I could pass a type mapper to process the response and possibly some options (eg. safe requests or a patch flag and so on)